text-generation-inference/integration-tests/conftest.py

```diff
@@ -99,58 +167,112 @@ def docker_launcher(
 
         try:
             container = client.containers.get(container_name)
+            logger.info(f"Stopping existing container {container_name}")
             container.stop()
             container.wait()
         except NotFound:
             pass
+        except Exception as e:
+            logger.error(f"Error handling existing container: {str(e)}")
 
-        env = {"LOG_LEVEL": "info,text_generation_router=debug"}
+        env = {
+            "LOG_LEVEL": "info,text_generation_router,text_generation_launcher=debug",
+            "MAX_BATCH_SIZE": "4",
+            "HF_HUB_ENABLE_HF_TRANSFER": "0",
+            "JETSTREAM_PT": "1",
```

Comment:
I moved it to a model config dict, so the arguments can be added for each models. I prefer to leave keep log_level as it allows for easier debugging in case a test fails